provisioning/device/samples/register_x509.js

```diff
@@ -5,7 +5,13 @@
 
 var fs = require('fs');
 var Transport = require('azure-iot-provisioning-device-http').Http;
+
+// Feel free to change the preceding using statement to anyone of the following if you would like to try another protocol.
```

Comment:
>preceding using statement to anyone [](start = 27, length = 35)

preceeding ?!using!? statement to any of the following

The using doesn't belong.
Might change it to something about commenting the previous statement and uncommenting one of the subsequent statements to try another protocol.

Reply:
supposed to be 'require'. fixed.

---
In reply to: [157665622](https://github.com/Azure/azure-iot-sdk-node/pull/184#discussion_r157665622) [](ancestors = 157665622)